src/axolotl/loaders/model.py

```
@@ -375,12 +376,21 @@ def _load_adapters(self) -> PeftConfig | None:
                 and self.cfg.rl in [RLType.DPO, RLType.IPO, RLType.KTO]
                 and not self.cfg.merge_lora
             ):
-                _, lora_config = load_lora(
-                    self.model, self.cfg, inference=False, config_only=True
-                )
+                try:
+                    builder = AdapterBuilderFactory.create_builder(
+                        self.cfg.adapter, self.cfg
+                    )
+                    lora_config = builder.build_config(self.model)
+                except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)